lab5/lab5_task_1.css

```diff
@@ -0,0 +1,410 @@
+main {    
+    display: block;
+    margin-left: auto; 
+    margin-right: auto;
+}
+
+input {
+    display: none;
+}
+
+label[for="view-list"] {
+    position: fixed; 
+    left: auto; 
+    right: 10px; 
+    top: 57px;
+}
+
+label[for="view-table"] {
+    position: fixed; 
+    left: auto; 
+    right: 10px; 
+    top: 22px;
+}
+
+input#view-list:checked + label {
+    border: 1px solid red;
+}
+
+input#view-table:checked + label {
+    border: 1px solid red;
+}
+
+input#view-list:checked ~ .item {
+    float: left;
+    width: 90%;
+    height: auto;
+    margin: 30px;
+    padding: 0px;
+    font-size: 14px;
+}
+
+
+#view-table:checked ~ .item {
+    float: left;
+    width: 300px;
+    height: 600px;
+    margin: 30px;
+    padding: 0px;
+    font-size: 14px;
+}
+.item {
+    border: 4px solid white;
+}
+
+.item:hover {
+    border: 4px solid lightcoral;
+}
+
+.item-description {    
+    float: left;
+    width: auto;
+    max-width: 500px;
+    line-height: 25px;
+    text-align: justify;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    margin: 0px;   
+}
+
+.item-description::before {
+    content: 'Описание товара: ';
+    color: lightseagreen;
+}
+
+.item-class {
+    color: grey;
+}
+
+.item-class::before {
+    content: 'категория: ';
+    color: lightseagreen;
+}
+
+figure {
+    display: block;
+    float: left;
+    height: 150px;
+    width: 280px;
+}
+
+figure > img {
+    display: block;
+    float: left;
+    height: 150px;
+    margin: 0px;    
+    bottom: 0px;   
+}
+
+.ncsp {
+    width: 280px;
+    float: left;    
+    padding: 5px;
+    margin: 0px;
+} 
+
+a {
+    display: inline-block;
+    height: 15px;
+    width: auto;
+    max-width: 300px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+
+fieldset {
+    border: none;
+}
+
+.item-condition1:not(:checked) > input {
+    position:absolute;
+    top:-9999px;
+    clip:rect(0,0,0,0);
```

Comment:
Try to be consistent in your CSS formatting
https://github.com/airbnb/css#formatting